validation-test/Driver/Dependencies/rdar25405605.swift

```diff
@@ -28,15 +28,15 @@
 
 // CHECK-1-SKIPPED-NOT: warning
 // CHECK-1-SKIPPED: {{^{$}}
-// CHECK-1-SKIPPED: "kind": "skipped"
-// CHECK-1-SKIPPED: "name": "compile"
-// CHECK-1-SKIPPED: ".\/main.swift"
+// CHECK-1-SKIPPED-DAG: "kind"{{ ?}}: "skipped"
+// CHECK-1-SKIPPED-DAG: "name"{{ ?}}: "compile"
+// CHECK-1-SKIPPED-DAG: "{{(\.\\\/)?}}main.swift"
 // CHECK-1-SKIPPED: {{^}$}}
 
 // CHECK-1-SKIPPED: {{^{$}}
-// CHECK-1-SKIPPED: "kind": "skipped"
-// CHECK-1-SKIPPED: "name": "compile"
-// CHECK-1-SKIPPED: ".\/helper.swift"
+// CHECK-1-SKIPPED-DAG: "kind"{{ ?}} "skipped"
```

Comment:
```suggestion
// CHECK-1-SKIPPED-DAG: "kind"{{ ?}}: "skipped"
```
Add back accidentally removed `:`